src/main/java/essenmakanan/ingredient/Ingredient.java

```diff
@@ -1,4 +1,28 @@
 package essenmakanan.ingredient;
 
 public class Ingredient {
+    private String name;
+    private String qty;
```

Comment:
maybe spelling out in full would be clearer? some might misinterpret as "quality"

Reply:
good point, will change the name